package.json

```
@@ -26,7 +26,7 @@
     "lint-staged": "15.2.0",
     "postcss-flexbugs-fixes": "5.0.2",
     "postcss-preset-env": "7.7.2",
-    "prettier": "3.1.1",
+    "prettier": "^3.1.1",
```

Let's keep this pinned to 3.1.1 to ensure everyone gets the exact correct version of the package when they run `npm install`.

```suggestion
    "prettier": "3.1.1",
```